Improve UI (again)

Str4ky opened the pull request:
## Summary
I've made a FAR better UI upgrade that DOES NOT break the program this time, could you look into this please? This looks far better and I really want to have this UI in the program to have a nice final product :)

## Here's some screenshots:
The main page:

![screen1](https://github.com/user-attachments/assets/2dd3c7c9-812d-44d8-8f34-0599a3bef5a1)

In the settings:

![screen2](https://github.com/user-attachments/assets/6e584071-2f4d-474b-adb7-965f8f30262b)

Players boxes:

![screen3](https://github.com/user-attachments/assets/9d553b70-e075-468f-ab4b-1eaec7d52893)

Str4ky committed:
feat: Improve UI

PabloMK7 commented:
Looks great, but I'd prefer not having the master volume slider in the settings page, as that one is meant to be quickly adjustable.

Also the mic volume slider should be changed in a way that won't cause confusion with it being the output volume

Str4ky committed:
fix: logo scale and settings panel position

Str4ky committed:
fix: increase name length limit

Str4ky committed:
fix: mic slider position

Str4ky commented:
> Looks great, but I'd prefer not having the master volume slider in the settings page, as that one is meant to be quickly adjustable.
> 
> Also the mic volume slider should be changed in a way that won't cause confusion with it being the output volume

Changed the mc volume slider to here

![image](https://github.com/user-attachments/assets/c62fc990-bc4a-43dd-a25c-4dd5d0162494)

I just don't know what you meant by "Also the mic volume slider should be changed in a way that won't cause confusion with it being the output volume"

Also changed the Windows 11 cog logo to the 3ds system settings wrench one

![image](https://github.com/user-attachments/assets/66eb3b24-bdc3-4670-b5bd-8a9be3082566)

Str4ky committed:
fix: settings button logo

Str4ky committed:
Update MainScene.unity

Str4ky committed:
Update LiberationSans SDF - Fallback.asset

Str4ky committed:
fix: address bar width

Str4ky committed:
fix: texts position in settings

Str4ky committed:
Update README.md

Str4ky committed:
Update README.md

Str4ky committed:
Update README.md

PabloMK7 commented:
That is the mic volume, not the master volume. The one I meant is the output volume for the speakers.

Str4ky commented:
Ooooh I see, lemme change that real quick

Str4ky committed:
fix: master volume texts and position, added selected image for mic input

Str4ky committed:
Merge branch 'main' of https://github.com/Str4ky/CTGP7ProximityChatClient

Str4ky committed:
Update README.md

Str4ky commented:
> That is the mic volume, not the master volume. The one I meant is the output volume for the speakers.

Made the changes you asked

![image](https://github.com/user-attachments/assets/6123e713-280f-4592-a105-14175f47eba3)

![image](https://github.com/user-attachments/assets/d9a5a2bb-bf93-4938-bf1f-94b21e4bb7c4)

And also replaced the check icon in the dropdown to this

![image](https://github.com/user-attachments/assets/773e9240-4cc4-4b87-a25b-77696a24a082)

Str4ky committed:
feat: edit ui font

Str4ky committed:
Merge branch 'main' of https://github.com/Str4ky/CTGP7ProximityChatClient

Str4ky committed:
Update README.md

Str4ky commented:
Also changed the font to be close to the one MK7 uses

![image](https://github.com/user-attachments/assets/070fcedf-d731-4f32-b1fd-62fca58972e3)

Str4ky committed:
Update MainScene.unity

Str4ky committed:
Merge branch 'main' of https://github.com/Str4ky/CTGP7ProximityChatClient
